compilers/src/make.js

```diff
@@ -2,8 +2,8 @@ const fs = require('fs')
 
 const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('../../hostfile', '##COINHIVE DOMAINS\n', ()=>{})
 domains.forEach((domain) => {
-  fs.appendFile('../../hostfile', '0.0.0.0 ' + domain + '\n')
+  fs.appendFile('../../hostfile', '0.0.0.0 ' + domain + '\n', ()=>{})
 })
```

Comment:
You're missing a semicolon

Reply:
[It's](http://blog.izs.me/post/2353458699/an-open-letter-to-javascript-leaders-regarding) [totally](http://inimino.org/~inimino/blog/javascript_semicolons) [fine.](https://www.youtube.com/watch?v=gsfbh17Ax9I)